.werft/build.yaml

```diff
@@ -89,15 +80,6 @@ pod:
       mountPath: /mnt/secrets/monitoring-satellite-stackdriver-credentials
     - name: monitoring-satellite-preview-token
       mountPath: /mnt/secrets/monitoring-satellite-preview-token
-    - name: gcp-sa
```

Comment:
The secret is still used in several places (search for `/mnt/secrets/gcp-sa`), let's not delete it.